2021/36xxx/CVE-2021-36338.json

```diff
@@ -1,17 +1,70 @@
 {
```

Comment:
'https://www.dell.com/support/kbdoc/000194640 ' does not match '^(ftp|http)s?://\\S+$'